test/find_xml_test.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'test_helper'
+
+describe "find_xml_file" do
+  it "should return nil if there are no xml files" do
+    expect(find_xml_file('.')).must_be_nil
+  end
+
+  it "should return an xml file if it's in the folder" do
```

Comment:
Is "example.xml" the only xml file in your test data folder? Consider clarifying which xml file you are expecting it to find